app/controllers/questions_controller.rb

```diff
@@ -2,4 +2,19 @@ class QuestionsController < ApplicationController
   def index
     @questions = Question.all.order(created_at: :desc)
   end
-end
\ No newline at end of file
+
+  def new
+    @question = Question.new
+  end
+
+  def create
+    Question.create(title: question_params[:title], detail: question_params[:detail])
```

Comment:
次のように書いて動作しませんか？
```suggestion
    Question.create(question_params)
```

また，バリデーションエラー時にフラッシュやエラーメッセージ を出すようにしてみましょう。
（メッセージ投稿アプリのその2・その3などを参考にして下さい）